tox.ini

```diff
@@ -21,11 +21,6 @@ changedir = {toxinidir}/tests
 commands =
   stestr run {posargs}
 
-[testenv:py36]
-extras =
```

Comment:
I think we can add the extras to the root tesyenv here. Iirc, I split it out for py310 because matplotlib didn't work when we first added support for it. Although I guess py311 might have the same issue, it's worth Che though.

Reply:
Done